source/kkp/kkp.h

```diff
@@ -60,4 +60,7 @@ int kkp2(unsigned char *X, int *SA, int n,
 int kkp1s(unsigned char *X, int n, std::string SA_fname,
     std::vector<std::pair<int, int> > *F);
 
+int parse_phrase(unsigned char* X, int n, int i, int psv, int nsv,

```

Comment:
where is the function definition? Did you forget to push the changes to the .cpp file?

Reply:
that's an auxiliary routine already defined in kkp.cpp